README.md

```diff
@@ -321,23 +330,23 @@ var nvmlDeviceGetCount = nvmlDeviceGetCount_v1
 
 // updateVersionedSymbols()
 func updateVersionedSymbols() {
-	ret := nvml.Lookup("nvmlInit_v2")
-	if ret == SUCCESS {
-		nvmlInit = nvmlInit_v2
-	}
-	ret = nvml.Lookup("nvmlDeviceGetPciInfo_v2")
-	if ret == SUCCESS {
-		nvmlDeviceGetPciInfo = nvmlDeviceGetPciInfo_v2
-	}
-	ret = nvml.Lookup("nvmlDeviceGetPciInfo_v3")
-	if ret == SUCCESS {
-		nvmlDeviceGetPciInfo = nvmlDeviceGetPciInfo_v3
-	}
-	ret = nvml.Lookup("nvmlDeviceGetCount_v2")
-	if ret == SUCCESS {
-		nvmlDeviceGetCount = nvmlDeviceGetCount_v2
-	}
-	...
+    ret := nvml.Lookup("nvmlInit_v2")
+    if ret == SUCCESS {
+        nvmlInit = nvmlInit_v2
+    }
+    ret = nvml.Lookup("nvmlDeviceGetPciInfo_v2")
+    if ret == SUCCESS {
+        nvmlDeviceGetPciInfo = nvmlDeviceGetPciInfo_v2
+    }
+    ret = nvml.Lookup("nvmlDeviceGetPciInfo_v3")
+    if ret == SUCCESS {
+        nvmlDeviceGetPciInfo = nvmlDeviceGetPciInfo_v3
+    }
+    ret = nvml.Lookup("nvmlDeviceGetCount_v2")
+    if ret == SUCCESS {
+        nvmlDeviceGetCount = nvmlDeviceGetCount_v2
+    }
+    ...
```

Comment:
I'd rather leave these as tabs since that is what `gofmt` produces if we run it through that.